packages/ui/primitives/document-flow/add-fields.tsx

```diff
@@ -340,7 +340,9 @@ export const AddFieldsFormPartial = ({
               <PopoverContent className="p-0" align="start">
                 <Command>
                   <CommandInput />
-                  <CommandEmpty />
+                  <CommandEmpty>
+                    <span>No Recipient matching this description was found.</span>
```

Comment:
Hi Peter!

Could you please add the following className to the span:
- `text-muted-foreground inline-block px-4`

And also lowercase Recipient?

Thank you!

Reply:
alright @dguyen 
Thanks

Reply:
@dguyen  updated